packages/lib/constants.ts

```diff
@@ -20,6 +20,7 @@ export const DEFAULT_BRAND_COLOR = "#64748b";
 export const FB_LOGO_URL =
   "https://s3.eu-central-1.amazonaws.com/listmonk-formbricks/Formbricks-Light-transparent.png";
 
+export const IS_LOCALHOST = ["localhost", "127.0.0.1", "::1"].includes(new URL(WEBAPP_URL).hostname);
```

Comment:
pls use the node_env "development" check here as its easier and then we won't need another constant at all 🙏